client/api/notebook.py

```diff
@@ -97,6 +101,21 @@ def save_notebook(self):
         else:
             print()
 
+def login_with_env(assignment):
+    access_token = os.environ.get('OKPY_ACCESS_TOKEN')
+    if not access_token:
+        log.info("ACCESS_TOKEN did not exist in the environment")
+        return
+    student_email = ok_auth.display_student_email(assignment.cmd_args, access_token)
+    if student_email:
+        # Token is valid,
+        expires_in = int(os.environ.get('OKPY_EXPIRES_IN', 60))
+        refresh_token = os.environ.get('OKPY_REFRESH_TOKEN')
```

Comment:
So I'm guessing it wouldn't really be useful as Kyle suggested for scripting if we rely on notebooks to set the `OKPY_EXPIRES_IN` and `OKPY_REFRESH_TOKEN` environment variables.

Is there a danger in just using the `OKPY_ACCESS_TOKEN` and not relying on the ok storage?  Part of the reason why I'm stuck merging this is I'm not sure how to address the OKPY_ACCESS_TOKEN-storage token potential conflict